.secrets.example

```diff
@@ -0,0 +1,14 @@
+LIT_NETWORK=datil-test
```

Comment:
If we have a secrets example why do we gen the `.env` file in CI? Perhaps we copy this file and override values?

Reply:
Each example needs its env file for its working and tests. I can update the action to copy `.secret.example` to each of `.env` files but we still need 4 private secrets to be kept in GitHub secrets and then update the env file with more specific logic.

Reply:
I see what you mean. I think for readability in CI it would be a nice refactor. But up to you